src/navigators/TabBar.tsx

```diff
@@ -260,57 +262,35 @@ const defaultStyles = createStyles('TabBar', (theme) => ({
   },
   labelActiveText: {
     alignSelf: 'center',
+    color: theme.colors.onSurface,
   },
-  labelActiveText0: {},
-  labelActiveText1: {},
-  labelActiveText2: {},
   labelInactiveText: {
     alignSelf: 'center',
+    color: theme.colors.onSurfaceDisabled,
   },
-  labelInactive0Text: {},
-  labelInactive1Text: {},
-  labelInactive2Text: {},
   activeIndicatorView: {
     position: 'absolute',
-    top: -6,
     height: 2 * 3,
     zIndex: 1,
-  },
-  activeIndicator0View: {
-    backgroundColor: theme.colors.primary,
-  },
-  activeIndicator1View: {
-    backgroundColor: theme.colors.secondary,
-  },
-  activeIndicator2View: {
-    backgroundColor: theme.colors.tertiary,
+    backgroundColor: theme.colors.scrim,
```

Comment:
Changing this color to the theme's `scrim` seems like a change (beyond just the refactoring to not expect only 3 tabs).  Is that correct?